src/database/sqlite3/sqlite_database.cc

```diff
@@ -35,88 +35,88 @@
 
 #include "config/config_manager.h"
 
-#define DB_BACKUP_FORMAT "{}.backup"
+static constexpr auto DB_BACKUP_FORMAT = "{}.backup";
 
 // updates 1->2
-#define SQLITE3_UPDATE_1_2_1 "DROP INDEX mt_autoscan_obj_id"
-#define SQLITE3_UPDATE_1_2_2 "CREATE UNIQUE INDEX mt_autoscan_obj_id ON mt_autoscan(obj_id)"
-#define SQLITE3_UPDATE_1_2_3 "ALTER TABLE \"mt_autoscan\" ADD \"path_ids\" text"
+static constexpr auto SQLITE3_UPDATE_1_2_1 = "DROP INDEX mt_autoscan_obj_id";
+static constexpr auto SQLITE3_UPDATE_1_2_2 = "CREATE UNIQUE INDEX mt_autoscan_obj_id ON mt_autoscan(obj_id)";
+static constexpr auto SQLITE3_UPDATE_1_2_3 = R"(ALTER TABLE "mt_autoscan" ADD "path_ids" text)";
 
 // updates 2->3
-#define SQLITE3_UPDATE_2_3_1 "ALTER TABLE \"mt_cds_object\" ADD \"service_id\" varchar(255) default NULL"
-#define SQLITE3_UPDATE_2_3_2 "CREATE INDEX mt_cds_object_service_id ON mt_cds_object(service_id)"
+static constexpr auto SQLITE3_UPDATE_2_3_1 = R"(ALTER TABLE "mt_cds_object" ADD "service_id" varchar(255) default NULL)";
+static constexpr auto SQLITE3_UPDATE_2_3_2 = "CREATE INDEX mt_cds_object_service_id ON mt_cds_object(service_id)";
 
 // updates 3->4: Move to Metadata table
-#define SQLITE3_UPDATE_3_4_1 "CREATE TABLE \"mt_metadata\" ( \
-  \"id\" integer primary key, \
-  \"item_id\" integer NOT NULL, \
-  \"property_name\" varchar(255) NOT NULL, \
-  \"property_value\" text NOT NULL, \
-  CONSTRAINT \"mt_metadata_idfk1\" FOREIGN KEY (\"item_id\") REFERENCES \"mt_cds_object\" (\"id\") \
-  ON DELETE CASCADE ON UPDATE CASCADE )"
-#define SQLITE3_UPDATE_3_4_2 "CREATE INDEX mt_metadata_item_id ON mt_metadata(item_id)"
+static constexpr auto SQLITE3_UPDATE_3_4_1 = R"(CREATE TABLE "mt_metadata" ()"
+                                             R"("id" integer primary key, )"
```

Comment:
Dont these quotes cause issues?

Reply:
My understanding is no. This whole thing is supposed to be on one line I think.